ChangeLog.md

```diff
@@ -3,6 +3,309 @@
 Notable changes to the FreeOrion project will be documented in this file.
 
 
+## [v0.4.6] - 2016-09-10
+
+
+### Key Changes
+
+- Reworked supply mechanics
+- Various GUI improvements
+ - GUI responsiveness / FPS improvements
+ - Made GUI more configurable
+ - Lot of other small improvements to handling/display of things
+- Turn processing time improvements
```

Comment:
Well, maybe the line doesn't convey sufficiently just how substantial the improvements are, so maybe rephrase it like this: `Substantial improvements to GUI responsiveness and FPS`? That ok for both of you?